apps/frontend/static_src/js/main.js

```diff
@@ -1 +1,63 @@
 import { Dropdown } from 'bootstrap';
+const debounce = require('lodash.debounce')
```

Comment:
Let's use the newer `import` syntax here:

```suggestion
import debounce from 'lodash.debounce';
```